addons/account/views/account_view.xml

```diff
@@ -90,6 +95,8 @@
                     <filter string="Expenses" name="expensesacc" domain="[('internal_group','=', 'expense')]"/>
                     <separator/>
                     <filter string="Account with Entries" name="used" domain="[('used', '=', True)]"/>
+                    <filter string="Active Account" name="activeacc" domain="[('deprecated', '=', False)]"/>
+                    <filter string="Account Deprecated" name="deprecatedacc" domain="[('deprecated', '=', True)]"/>
```

Comment:
I don't think a filter showing only deprecated accounts is relevant. IMO, you can remove this line.

Reply:
Wait @avw-odoo